### PR TITLE
[CHORE] Switch to react-native-localize

### LINKED ISSUE #899: Switch to react-native-localize
react-native-i18n is [deprecated](https://github.com/AlexanderZaytsev/react-native-i18n#%EF%B8%8F-this-package-is-deprecated) and we need to switch to the [recommended lib from RN community](https://github.com/react-native-community/react-native-localize)

### PR DESCRIPTION
**Pull request by djorkaeffalexandre**
@RocketChat/ReactNative

Closes #899

### CODE REVIEW

File: app/presentation/RoomItem/index.js [hunk 1]
```diff
@@ -320,12 +320,17 @@ export default class RoomItem extends React.Component {
 		);
 	}
 
+	capitalize = (s) => {
```
Comment (diegolmello) [inline app/presentation/RoomItem/index.js]: Can you move `capitalize` and `formatDate` away from the component?

File: app/views/RoomsListView/index.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { connect } from 'react-redux';
 import { isEqual } from 'lodash';
 import { SafeAreaView } from 'react-navigation';
 import Orientation from 'react-native-orientation-locker';
+import moment from 'moment';
+import 'moment/min/locales';
```
Comment (diegolmello) [inline app/views/RoomsListView/index.js]: Is this needed?

Comment (djorkaeffalexandre) [reply app/views/RoomsListView/index.js]: yes :(
https://github.com/moment/moment/issues/4422

File: app/presentation/RoomItem/index.js (hunk 1 above)
Comment (djorkaeffalexandre) [reply app/presentation/RoomItem/index.js]: fixed.

**Pull request merged**